src/map.cpp

```diff
@@ -7853,9 +7843,9 @@ void map::draw_rough_circle_ter( ter_id type, int x, int y, int rad )
     }, x, y, rad );
 }
 
-void map::draw_rough_circle_ter( std::string type, int x, int y, int rad )
+void map::draw_rough_circle_ter( ter_str_id &type, int x, int y, int rad )
```

Comment:
I've done it already (as a  part of 'Other code improvements'), but didn't push the unfinished commit yet.